handbook/company/pricing-features-table.yml

```diff
@@ -765,6 +771,7 @@
   tier: Free
   jamfProHasFeature: yes
   jamfProtectHasFeature: yes
+  isExperimental: yes
```

Comment:
```suggestion
```

Not even sure Fleet can do this today. I think we can remove this one from the pricing page unless I'm missing something.

Reply:
UPDATE: I see we have the comingSoonOn in 2025. I think we can leave this one in but remove the `isExperimental`